### PR TITLE
Contribution and code of conduct

### PR DESCRIPTION
**Pull request by Maar-io**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (bobo-k2) [inline CODE_OF_CONDUCT.md:64]: Are you planning to insert this later?

**Review by bobo-k2 — APPROVED** [review]
Beautiful

**Review by midegdugarova — APPROVED** [review]
All looks good to me except inserting the contact detail

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Maar-io) [reply CODE_OF_CONDUCT.md:64]: do you have any suggestion?

**Comment by Maar-io** [comment]
> All looks good to me except inserting the contact detail

do you have any suggestion who can be contact?

File: profile/README.md [hunk 2]
```diff
@@ -3,3 +3,37 @@
...
+| [swanky-cli](https://github.com/swankyhub/swanky-cli)  | tool  | -  | [README](https://github.com/swankyhub/swanky-cli/blob/master/README.md)  |   |
+| [swanky-node](https://github.com/swankyhub/swanky-node)  | tool  | -  | [README](https://github.com/swankyhub/swanky-node/blob/main/README.md)  |   |
+| [dex]()   | ink!  | -  | [tutorial](https://docs.astar.network/docs/build/wasm/from-zero-to-ink-hero/dex/)  |   |
+| [nft]()   | ink!  | -  | [tutorial](https://docs.astar.network/docs/build/wasm/from-zero-to-ink-hero/nft/)  |   |
+| [manic-minter ]()  | ink!  | -  | [tutorial](https://docs.astar.network/docs/build/wasm/from-zero-to-ink-hero/manic-minter/)  |   |
+| [rmrk-lazy-mint]()   | ink!  | -  | -  |   |
+| [meta-transaction](https://github.com/swankyhub/meta-transaction)   | ink!  | -  | -  |   |
+| [marketplace](https://github.com/swankyhub/marketplace)   | ink!  | -  | -  |   |
+| [lottery](https://github.com/swankyhub/wasm-lottery)  | tool  | ✅  | -  |   |
+| [farming](https://github.com/swankyhub/farming)  | tool  | -  | -  |   |
```
Comment (codespool) [inline profile/README.md:26]: I don't think tool and ink! are the same category, could it be something like tool/example?
and ink goes to language category or something like that?
Anyway, lottery and farming should probably be of that (not tool) category.

Comment (Maar-io) [reply profile/README.md:26]: fixed

**Pull request merged**